tasks/envrc-template

```diff
@@ -1,37 +1,36 @@
-HOSTNAME=$BOSH_LITE_NAME.$DOMAIN_NAME
+BOSHLITE_HOSTNAME=$BOSH_LITE_NAME.$DOMAIN_NAME
 
 function exit_with_error_message() {
   echo "Error - /etc/hosts is missing the following entry:"
-  echo "$(cat ip) $HOSTNAME"
+  echo "$(cat ip) ${BOSHLITE_HOSTNAME}"
   exit 1
 }
 
-set +e
-grep --quiet $HOSTNAME /etc/hosts
+grep --quiet ${BOSHLITE_HOSTNAME} /etc/hosts
 HOST_FOUND=$?
-set -e
 if [ $HOST_FOUND -ne 0 ]; then
   exit_with_error_message
 else
-  set +e
-  grep $HOSTNAME /etc/hosts | grep --quiet $(cat ip)
+  grep ${BOSHLITE_HOSTNAME} /etc/hosts | grep --quiet $(cat ip)
   CORRECT_IP=$?
-  set -e
   if [ $CORRECT_IP -ne 0 ]; then
     exit_with_error_message
   fi
 fi
 
-echo "Please make sure the host $HOSTNAME is reachable"
+echo "Please make sure the host ${BOSHLITE_HOSTNAME} is reachable"
 export BOSH_ENVIRONMENT="$DIRECTOR_URL"
 export BOSH_CA_CERT="$CA_CERT"
 export BOSH_CLIENT=admin
 export BOSH_CLIENT_SECRET="$CLIENT_SECRET"
 
 chmod 600 jumpbox.key
 
-ssh-keygen -q -R "$HOSTNAME" > /dev/null
+mkdir -p ~/.ssh
+test -f ~/.ssh/known_hosts || touch ~/.ssh/known_hosts
```

Comment:
why not just touch it? touch is more or less a no-op if the file exists.

Reply:
you are right, that makes it even simpler
I have updated the PR (force pushed)